components/camel-as2/camel-as2-component/src/main/java/org/apache/camel/component/as2/AS2Component.java

```diff
@@ -58,8 +58,8 @@ protected Endpoint createEndpoint(String uri, String methodName, AS2ApiName apiN
     }
 
     @Override
-    protected void doStart() throws Exception {
-        super.doStart();
+    protected void doInit() throws Exception {
+        super.doInit();
         if (Security.getProvider("BC") == null) {
```

Comment:
I wonder if this may cause some problems, as its in the JDK itself and if your build vs runtime are not the same then it can cause problems. I think we need to make this part in the doStart

Reply:
yeah, it makes sense.